### PR TITLE
Improve travis

### PR DESCRIPTION
**Pull request by lmignon**
I've simplified travis and setup.py to only use standard python tools.

### CODE REVIEW

**Comment by sbidoul** [comment]
:+1: I also like tox because it's the simplest way to explain how to run tests locally (ie just type `tox`), and the same tests run on travis.

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
-  apt:
-    packages:
-      - expect-dev
+cache: pip
 
 language: python
 
 python:
   - "2.7"
-  - "3.4"
```
Comment (lasley) [inline .travis.yml:10]: Why removing 3.4 and 3.6? I use 3.6 personally

**Comment by lasley** [comment]
Thanks for submitting this @lmignon - I finally have an example of how to use Tox. I don't even want to admit how long I tried to get this working a few years back.

File: .travis.yml (hunk 1 above)
Comment (lmignon) [reply .travis.yml:10]: @lasley because Odoo officially supports 2.7 and 3.5.  :smirk:

Comment (lasley) [reply .travis.yml:10]: Is there harm in leaving 3.6? 

They only officially support the outdated wkhtmltopdf v0.12.1 too, but I can attest that 0.12.4 works perfectly and does not leave my system insecure like the earlier version. I find this similar situation

Comment (lmignon) [reply .travis.yml:10]: @lasley I don't know... For the lib itself, it seems safe.

Comment (sbidoul) [reply .travis.yml:10]: Odoo's commitment is >=3.5 even if they test only 3.5. So keep 3.6.

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+# and then run "tox" from this directory.
+
+[tox]
+envlist = 
+  py27
+  py35
+
+[testenv]
+usedevelop = True
+commands = py.test --cov-config=.coveragerc --cov=oca --flake8 {posargs}
```
Comment (sbidoul) [inline tox.ini]: Also nice to have is `--cov-branch`

File: .travis.yml (hunk 1 above)
Comment (lasley) [reply .travis.yml:10]: We'll also need to remove the 3.4 classifier from [setup.py](https://github.com/LasLabs/oca-decorators/pull/1/files#diff-2eeaed663bd0d25b7e608891384b7298L29)

If the commitment is >=3.5, should we think about adding `3.7-dev` to Travis and `3.7` to the setup identifiers?

Comment (sbidoul) [reply .travis.yml:10]: @lasley 

> leave my system insecure like the earlier version

Unrelated but can you share any pointer about those security issues?

Comment (lmignon) [reply .travis.yml:10]: @lasley The 3.4 classifier is removed from setup.py

Comment (lasley) [reply .travis.yml:10]: If you're using static binaries (or Windows), OpenSSL and Libpng come bundled. Assuming you're using 0.12.1.2, the OpenSSL is [1.0.1j](https://www.cvedetails.com/vulnerability-list/vendor_id-217/product_id-383/version_id-178264/Openssl-Openssl-1.0.1j.html)

Comment (lasley) [reply .travis.yml:10]: If you're default *nix installation you're fine FYI. It's dynamic by default AFAIK

**Comment by lasley** [comment]
Thanks @lmignon

**Pull request merged**